nova/compute/instance_actions.py

```diff
@@ -75,3 +75,5 @@
 # nova-manage instance actions logged to allow operators and users alike to
 # track out of band changes made to their instances.
 NOVA_MANAGE_REFRESH_VOLUME_ATTACHMENT = 'refresh_volume_attachment'
+
+SAP_IN_CLUSTER_VMOTION = 'in_cluster_vmotion'
```

Comment:
Wouldn't it make sense to also prefix the string with `sap_`?.
I.e.
```
SAP_IN_CLUSTER_VMOTION = 'in_cluster_vmotion'
```
I understand that the string matches the function, but it would reduce the risk of a collision of the data in db (which is worse than the function).
Not that I consider it likely, or that I would block this PR because of that.

Reply:
Since this is a customer-facing string, I didn't want to add the `sap` in it arguing the same unlikeliness of upstream ever having a cluster.
The action in the API is also called `in_cluster_vmotion` - albeit in the `/sap/` namespace of the API.

I guess I'll just change it. Nobody will complain and we're on the save side.